components/hab/src/command/studio.rs

```diff
@@ -59,6 +65,23 @@ pub fn start(ui: &mut UI, args: Vec<OsString>) -> Result<()> {
     inner::start(ui, args)
 }
 
+/// Retrieves the
+pub fn image_identifier() -> String {
+    let version: Vec<&str> = VERSION.split("/").collect();
```

Comment:
Will `VERSION` always contain a `/`?  If it doesn't, does this do the right thing?